tests/by-util/test_df.rs

```diff
@@ -28,16 +28,9 @@ fn test_df_compatible_si() {
 
 #[test]
 fn test_df_output() {
```

Comment:
I think we should change the name of this test to something more descriptive because it doesn't test the output anymore.